packages/flutter_tools/lib/src/android/aar.dart

```diff
@@ -0,0 +1,46 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:meta/meta.dart';
+
+import '../base/common.dart';
+import '../build_info.dart';
+import '../project.dart';
+
+import 'android_sdk.dart';
+import 'gradle.dart';
+
+/// Builds the AAR and POM files for the current Flutter module or plugin.
+Future<void> buildAar({
```

Comment:
According to https://codecov.io/gh/flutter/flutter/pull/35217/diff#D16-17, this method has no test coverage.